adr/0011-sap-system-database-aggregate-split.md

```diff
@@ -0,0 +1,33 @@
+# 11. SAP system and database aggregate split
+
+Date: 2024-04-04
+
+## Status
+
+Accepted
+
+## Context
+
+In the initial implementation of Trento, the SAP system and database states are stored in the same `sap_system.ex` aggregate. This makes the assumption that the relationship between a SAP system and a database is of 1 to 1, meaning that a database is only used by one SAP system. This is not an accurate description of the reality, as a database can handle multiple SAP systems in a multi-tenant environment.
```

Comment:
Do we need the 'of' in the 1 to 1 sentence? 

> a database is of 1 to 1 ...

Maybe just 
> a database is 1 to 1 ...

Reply:
I don't know. For me both look correct. The one with `of` looks more accurate though

Reply:
no actually Eugen is right, I think you're translating literally from Spanish (I guess because it would be similar in Italian) but in English you need to drop the "of".